week-3/task2/WithTimeTest.js

```diff
@@ -0,0 +1,18 @@
+import WithTime from "./WithTime.js";
+
+const fetchFromUrl = async (url, cb) => {
+    const response = await fetch(url);
+    const posts = await response.json();
```

Comment:
```suggestion
const fetchFromUrl = (url, cb) => {
    const response = fetch(url);
    const posts = response.json();
```
You should implement this without change the function to async. In this case you should use the `cb` (callback) function to return the data.